srcpkgs/limine/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'limine'
+pkgname=limine
+version=2.24
+revision=1
+archs="x86_64* i686*"
+hostmakedepends="which wget nasm mtools git tar"
```

Comment:
The way this package uses git is a no-go. git repos should never be downloaded at build time.
Similiar with `wget`, every file fetched should preferably be in distfiles and have a checksum, the build scripts do no even try to verify if a downloaded file is correct/didn't change.

Reply:
Ideally since they download over `https` there should be no malicious tampering done at download time. It would be possible to download the packages needed using distfiles ahead of time anyways, will do that.

Reply:
This isn't mainly about mailicous, but about reproducibility and such. It isn't that uncommon that someone randomly changes a tag or so.

Reply:
Also using `distfiles` allows us to cache the file, so it doesn't need to be downloaded for every run for every arch.